internal/projects/retrieve.go

```diff
@@ -51,33 +50,23 @@ func (c *Client) Retrieve() *cobra.Command {
 				return fmt.Errorf("Must specify only one of project-id and project name")
 			}
 
-			inc := []string{}
-
-			// only fetch extra details when rendered
-			switch c.Servicer.Format() {
-			case outputs.FormatJSON, outputs.FormatYAML:
-				inc = append(inc, "members")
-			}
-
-			listOpts := c.Servicer.ListOptions(inc, nil)
-
 			if projectID == "" {
-				projects, _, err := c.ProjectService.List(listOpts)
+				projects, _, err := c.ProjectService.FindProjects(context.Background()).Execute()
```

Comment:
- The `listOpts` need to be converted to `Includes`, otherwise we're not making the same API calls
- You'll need to use the `getAllProjects` function that was added for the `init` command in order to handle pagination

Reply:
updated as per comment